x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -669,7 +669,7 @@ public void testTopHitsAggregationWithTwoArgs() {
             assertTrue(eqe.output().get(0).dataType() == DataType.KEYWORD);
             assertThat(eqe.queryContainer().aggs().asAggBuilder().toString().replaceAll("\\s+", ""),
                 endsWith("\"top_hits\":{\"from\":0,\"size\":1,\"version\":false,\"seq_no_primary_term\":false," +
-                    "\"explain\":false,\"docvalue_fields\":[{\"field\":\"keyword\",\"format\":\"use_field_mapping\"}]," +
+                    "\"explain\":false,\"docvalue_fields\":[{\"field\":\"keyword\"}]," +
                     "\"sort\":[{\"int\":{\"order\":\"desc\",\"missing\":\"_last\",\"unmapped_type\":\"integer\"}}," +
                     "{\"keyword\":{\"order\":\"desc\",\"missing\":\"_last\",\"unmapped_type\":\"keyword\"}}]}}}}}"));
```

Comment:
I cannot wrap my head around `missing` option for top_hits in the context of first/last/min/max... It's about how `null`/missing values are treated when sorting: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html#_missing_values. Is it ok to keep `missing` with the default (`_last`), or I'm missing something, or there are no tests that have `missing: _first` scenario included...?

Reply:
After I clicked "add comment" I realized these are about first/last non-NULL values. So yeah, we don't care about `missing` and we like the fact that the default is `_last`.